test.js

```diff
@@ -214,3 +214,29 @@ test('respects gitignore option false - sync', t => {
 	const actual = m.sync('*', {gitignore: false, onlyFiles: false});
 	t.true(actual.indexOf('node_modules') > -1);
 });
+
+// https://github.com/sindresorhus/globby/issues/105
+test.failing('throws ENOTDIR when specifying a file as cwd - async', async t => {
+	t.plan(1);
+
+	const isFile = path.resolve('fixtures/gitignore/bar.js');
+
+	try {
+		await m('.', {cwd: isFile});
+	} catch (error) {
+		t.is('ENOTDIR', error.code);
+	}
+});
+
+// https://github.com/sindresorhus/globby/issues/105
+test.failing('throws ENOTDIR when specifying a file as cwd - sync', t => {
+	t.plan(1);
+
+	const isFile = path.resolve('fixtures/gitignore/bar.js');
+
+	try {
+		m.sync('.', {cwd: isFile});
+	} catch (error) {
+		t.is('ENOTDIR', error.code);
+	}
```

Comment:
Use `t.throws` instead of try/catch. Then you can drop the `t.plan()`.

Reply:
Done.